adapters/adapterstest/test_json.go

```diff
@@ -318,6 +321,15 @@ func diffHttpRequests(description string, actual *adapters.RequestData, expected
 			return err
 		}
 	}
+
+	if len(expected.ImpIDs) < 1 {
+		return fmt.Errorf(`expected.ImpIDs must contain at least one imp ID`)
+	}
+
+	opt := cmpopts.SortSlices(func(a, b string) bool { return a < b })
+	if !cmp.Equal(expected.ImpIDs, actual.ImpIDs, opt) {
+		return fmt.Errorf(`%s actual.ImpIDs "%q" do not match expected "%q"`, description, actual.ImpIDs, expected.ImpIDs)
+	}
```

Comment:
Please use testify assert instead of pulling in a new package.

Reply:
In `diffHttpRequests` where we assert RequestData, we don't have access of `t *testing.T` so can't use `assert.ElementsMatch`, I guess that's the reason why @bsardo suggested assertion of `impIDs` using `cmp`.

Reply:
Yes that's the reason I suggested this approach. I explored using `assert.ElementsMatch` but it requires a larger rework of this file which I think should be handled in a separate PR at some point.